cds/modules/deposit/api.py

```diff
@@ -55,3 +58,189 @@ def create(cls, data, id_=None):
     def multipart_files(self):
         """Get all multipart files."""
         return MultipartObject.query_by_bucket(self.files.bucket)
+
+
+def video_resolver(ids):
+    """Get records from PIDs."""
+    pids = [p.object_uuid for p in PersistentIdentifier.query.filter(
+        PersistentIdentifier.pid_value.in_(ids)).all()]
+    return Video.get_records(pids)
+
+
+def deposit_build_url(video_id):
+    """Build video url."""
+    return url_for('invenio_deposit_ui.depid', pid_value=video_id)
```

Comment:
it looks like that for Flask doesn't make difference.
I already have a [test](https://github.com/CERNDocumentServer/cds/pull/254/files#diff-1c699497e5aaa8ed7b979398323dc496R59).
But I'll add also this for security: 
```
assert '/deposit/1' == deposit_build_url('1')
```